docs/source/rule-testing.rst

```diff
@@ -185,7 +192,14 @@ Integration tests can be restricted to **specific rules** to reduce time and out
 
 .. code-block:: bash
 
-  $ python manage.py lambda test --processor rule --rules <rule_01> <rule_02>
+  $ python manage.py lambda test --processor rule --test-rules <rule_01> <rule_02>
+
+Integration tests can be restricted to **specific file names** to reduce time and output (the extension is optional):
```

Comment:
I had to re-read "the extension is optional" multiple times to try and understand what this was talking about. Maybe `(the .json suffix is optional)`